master/buildbot/reporters/github.py

```diff
@@ -50,7 +50,10 @@ def reconfigService(self, token,
             baseURL = baseURL[:-1]
 
         self._http = yield httpclientservice.HTTPClientService.getService(
-            self.master, baseURL, headers={'Authorization': 'token ' + token})
+            self.master, baseURL, headers={
+                'Authorization': 'token ' + token,
+                'User-Agent': 'buildbot'
```

Comment:
Two comments here:

* it's `Buildbot`
* personally, I'd prefer to include the version